### PR TITLE
Adding ability to create files from contents

### PR DESCRIPTION
**Pull request by kwoodson**
I ran into an issue while passing too much data on the command line to the configmap literal argument.  I went ahead and allowed for the user to pass in content and we will write it to file in order to pass the data to the configmap.

I also added the subsequent clean up in an attempt to remove artifacts.

### CODE REVIEW

File: roles/openshift_daemonset_config/defaults/main.yml [hunk 1]
```diff
@@ -8,6 +8,7 @@ openshift_daemonset_config_monitoring_pos: "false"
 openshift_daemonset_config_node_selector:
   config: config
 openshift_daemonset_config_sa_name: configurator
+openshift_daemonset_config_configmap_contents_to_files: []
```
Comment (sdodson) [inline roles/openshift_daemonset_config/defaults/main.yml]: Can you make sure the README.md has an example of how to define this?

Comment (kwoodson) [reply roles/openshift_daemonset_config/defaults/main.yml]: @sdodson, I've added a README.md.

**Comment by kwoodson** [comment]
@sdodson, please check out the `README.md` when you get a chance.

**Comment by sdodson** [comment]
flakezilla

**Pull request merged**